### PR TITLE
Update/tracking

### PR DESCRIPTION
**Pull request by eshleebien**
-added travis
-track clicks
-changed error page

### CODE REVIEW

File: .env.travis [hunk 1]
```diff
@@ -0,0 +1,25 @@
+APP_ENV=local
+APP_DEBUG=true
+APP_KEY=PFnIKmvTvW8gsnhCMAiP4hjXRKq2Fq4l
```
Comment (wildtron) [inline .env.travis:3]: real key?

Comment (eshleebien) [inline .env.travis:3]: nope

Comment (eshleebien) [inline .env.travis:3]: I mean just a separate key from the live version.

Comment (wildtron) [inline .env.travis:3]: **resolved**

**Comment by wildtron** [comment]
this is php code base right? have you run `php -l` to all files just to make sure? :dancers: 

to all php files: `find -name '*.php' | xargs -n1 -d'\n' php -l`

**Comment by eshleebien** [comment]
I have a syntax checker in vim
https://gyazo.com/26c8f8c7f079df2701f1e4a82e50c444

**Comment by wildtron** [comment]
I see.. :up:

File: config/app.php [hunk 2]
```diff
@@ -28,6 +28,8 @@
 
     'url' => 'http://goto.tm',
 
+    'geoip' => 'https://www.telize.com/',
```
Comment (Jhoem) [inline config/app.php:31]: Is this being used or going to be used?

Comment (eshleebien) [inline config/app.php:31]: i will use this for tracking by country feature :)

**Comment by rycastaneda** [comment]
:up:

**Pull request merged**